src/app/components/organisms/Tabs/Tabs.sc.ts

```diff
@@ -79,6 +80,17 @@ export const TabHeader = styled.button<TabHeaderProps>`
     }
 `;
 
+export const TabHeaderText = styled.div`
+    /* stylelint-disable-next-line value-no-vendor-prefix */
+    display: -webkit-box;
+    max-height: 32px;
```

Comment:
Please use the theme.spacing function.
It's being used everywhere (afaik), so better stick to it.